src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1334,6 +1335,43 @@ public void checkOutFailsWhenEventPublishingFailsWithBadRequestError() {
     assertThat(response.getBody(), containsString(
       "Error during publishing Event Message in PubSub. Status code: 400"));
   }
+
```

Comment:
Is there any way in a test to simulate the act of overriding the blocks to prove that the checkout action can still proceed or is it out of scope?